### PR TITLE
add H_Quest Invitation_Cards

### PR DESCRIPTION
**Pull request by KnowOne134**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

adds the hidden quest for turning in the lure of the wildcat invitation cards.
they can be turned in anytime and skipped, you can turn in as many as you want until all 4 cards are done then its marked as complete.
you only get bonus coins on 1st turn in, after that its a static reward

## Steps to test these changes

add 1-4 kis on player and trigger Naja until you get right prompt to show cards
yes/no option
yes: collect reward
no: zone and come back to re engage convo

### CODE REVIEW

**Comment by KnowOne134** [comment]
not sure what to do about that error

File: scripts/quests/hiddenQuests/Invitation_Cards.lua [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            {
+                [74] = function(player, csid, option, npc)
+                    if option == 1 then
+                        local cards, coin, amount = quest.getCards(player)
+                        player:updateEvent({ [0] = cards, [1] = coin, [2] = amount, text_table = 0 })
+                    end
+                end,
+                [75] = function(player, csid, option, npc)
+                    if option == 1 then
+                        local cards, coin, amount = quest.getCards(player)
```
Comment (claywar) [inline scripts/quests/hiddenQuests/Invitation_Cards.lua]: `local cards, _, _ = quest.getCards(player)`

Comment (claywar) [reply scripts/quests/hiddenQuests/Invitation_Cards.lua]: If that doesn't work, just try assigning cards by itself since its the first returned value

**Comment by KnowOne134** [comment]
Says variables aren't used but they are, tested and returned cs data is correct

**Review by claywar — APPROVED** [review]

**Pull request merged**